models/account/account.go

```diff
@@ -0,0 +1,106 @@
+package account
+
+import (
+	"database/sql"
+	"fmt"
+	"strconv"
+
+	"github.com/b0d0nne11/scroll-test-project/db"
+	"github.com/mailgun/scroll"
+)
+
+type Account struct {
+	ID   int64
+	Name string
+}
+
+func New(name string) *Account {
+	return &Account{
+		Name: name,
+	}
+}
+
+func (a *Account) Save() error {
+	dbh, _ := db.Get()
```

Comment:
Then the application will panic on startup here, https://github.com/b0d0nne11/scroll-test-project/blob/brendan/devel/main.go#L50. And because `db.Get()` was written to always return the same handle and error if it succeed in the application startup it should never fail later. I can definitely see how this would be confusing so I addressed it while refactoring the DB module. `db.Get()` now only returns the DB handle.